amethyst_animation/src/resources.rs

```diff
@@ -42,10 +45,25 @@ where
     T: InterpolationPrimitive,
 {
     /// Time of key frames
+    ///
+    /// A simple example of this for animations that are defined with 4 evenly spaced key frames is
+    /// `vec![0., 1., 2., 3.]`.
     pub input: Vec<f32>,
     /// Actual output data to interpolate
+    ///
+    /// For `input` of size `i`, the `output` is size `n * i`, where `n` is the number of values for
```

Comment:
This is not completely correct, the size of the output can be anything the function requires, so there's no specific limit on this size.

Reply:
A summary of the predefined functions would be:

* Linear - i
* SphericalLinear - i
* CubicSpline - i * 3
* CatmullRomSpline - i + 2
* Step - i